core/src/main/java/io/grpc/internal/MutableHandlerRegistry.java

```diff
@@ -29,7 +29,9 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.grpc;
+package io.grpc.internal;
```

Comment:
NettyServerBuilder shouldn't be `@Internal`. It was expected that applications could implement their own. You should talk to @louiscryan before moving it to internal, since he was the main supporter of making it public.